packages/office-ui-fabric-react/src/components/Facepile/Facepile.base.tsx

```diff
@@ -58,7 +58,7 @@ export class FacepileBase extends BaseComponent<IFacepileProps, {}> {
         {this.onRenderAriaDescription()}
         <div className={_classNames.itemContainer}>
           {showAddButton ? this._getAddNewElement() : null}
-          <ul className={_classNames.members} role="listbox">
+          <ul className={_classNames.members} role="listbox" aria-label={'Facepile persona list'}>
```

Comment:
>Facepile persona list' [](start = 74, length = 22)

This needs to be a localized string, so it needs to be provided by the consumer of the component

Reply:
you're right @mtennoe,  thanks!